staging/src/k8s.io/apimachinery/pkg/util/validation/validation.go

```diff
@@ -67,6 +69,21 @@ func IsQualifiedName(value string) []string {
 	return errs
 }
 
+// IsFullyQualifiedName checks if the name is fully qualified.
+func IsFullyQualifiedName(fldPath *field.Path, name string) field.ErrorList {
```

Comment:
I renamed the function as it's used by the admission webhook validation as well.